ios/Classes/FlutterRTMPStreaming.swift

```diff
@@ -187,34 +187,34 @@ public class FlutterRTMPStreaming : NSObject {
 
 
 class MyRTMPStreamQoSDelagate: RTMPStreamDelegate {
-    let minBitrate: UInt32 = 300 * 1024
-    let maxBitrate: UInt32 = 2500 * 1024
+    let minBitrate: UInt32 = 6000 * 1024
```

Comment:
You can adjust this to a lower value (like `1000 * 1024`) if you want it to be further customizable. The increasing of the maxBitrate below is the important part to allow a 6000kbps rate.